src/rsz/test/pin_swap1.ok

```diff
@@ -767,9 +767,9 @@ sky130_fd_sc_hd__nor4_1/A -> B C D
 sky130_fd_sc_hd__nor4_1/B -> A C D
 sky130_fd_sc_hd__nor4_1/C -> A B D
 sky130_fd_sc_hd__nor4_1/D -> A B C
-sky130_fd_sc_hd__nor4_2/A -> B C D
-sky130_fd_sc_hd__nor4_2/B -> A C D
-sky130_fd_sc_hd__nor4_2/C -> A B D
+sky130_fd_sc_hd__nor4_2/A -> D B C
+sky130_fd_sc_hd__nor4_2/B -> D A C
+sky130_fd_sc_hd__nor4_2/C -> D A B
```

Comment:
Huh, my local run and the CI server disagree on the iteration order here. Are the tests flaky?

Reply:
It looks like the perennial problem of stl sets comparing pointers:
`typedef Set<LibertyPort*> LibertyPortSet;`
in
```
        sta::LibertyPortSet ports;
        equivCellPins(cell, port, ports);
        std::ostringstream ostr;
        for (auto port : ports) {
          ostr << ' ' << port->name();
        }
```